src/connector/src/base.rs

```diff
@@ -257,7 +258,23 @@ impl SplitMetaData for ConnectorState {
 pub enum ConnectorStateV2 {
     // ConnectorState should change to Vec<ConnectorState> because there can be multiple readers
     // in a source executor
-    State(ConnectorState),
+    // State(ConnectorState),
     Splits(Vec<SplitImpl>),
     None,
```

Comment:
Really I'd like to know what None means as a state. Empty state or?

Reply:
it's to represent no split is assigned to the executor.